### ISSUE TITLE
DeltaLog java.io.IOException: Filesystem closed

### ISSUE THREAD

**Issue by chongchuanbing** [delta-io/connectors#40]
when use hive jdbc,hive metastore operation error, the full stack trace:
```
Caused by: java.io.IOException: Filesystem closed
        at org.apache.hadoop.hdfs.DFSClient.checkOpen(DFSClient.java:817)
        at org.apache.hadoop.hdfs.DFSClient.getFileInfo(DFSClient.java:2114)
        at org.apache.hadoop.hdfs.DistributedFileSystem$22.doCall(DistributedFileSystem.java:1305)
        at org.apache.hadoop.hdfs.DistributedFileSystem$22.doCall(DistributedFileSystem.java:1301)
        at org.apache.hadoop.fs.FileSystemLinkResolver.resolve(FileSystemLinkResolver.java:81)
        at org.apache.hadoop.hdfs.DistributedFileSystem.getFileStatus(DistributedFileSystem.java:1301)
        at org.apache.spark.sql.delta.DeltaLogFileIndex$$anonfun$apply$1.apply(DeltaLogFileIndex.scala:56)
        at org.apache.spark.sql.delta.DeltaLogFileIndex$$anonfun$apply$1.apply(DeltaLogFileIndex.scala:56)
        at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
        at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
        at scala.collection.immutable.List.foreach(List.scala:392)
        at scala.collection.TraversableLike$class.map(TraversableLike.scala:234)
        at scala.collection.immutable.List.map(List.scala:296)
        at org.apache.spark.sql.delta.DeltaLogFileIndex$.apply(DeltaLogFileIndex.scala:56)
        at org.apache.spark.sql.delta.DeltaLog$$anonfun$org$apache$spark$sql$delta$DeltaLog$$updateInternal$1$$anonfun$apply$7.apply(DeltaLog.scala:313)
        at org.apache.spark.sql.delta.DeltaLog$$anonfun$org$apache$spark$sql$delta$DeltaLog$$updateInternal$1$$anonfun$apply$7.apply(DeltaLog.scala:275)
        at org.apache.spark.sql.delta.util.DeltaProgressReporter$class.withStatusCode(DeltaProgressReporter.scala:30)
        at org.apache.spark.sql.delta.DeltaLog.withStatusCode(DeltaLog.scala:58)
        at org.apache.spark.sql.delta.DeltaLog$$anonfun$org$apache$spark$sql$delta$DeltaLog$$updateInternal$1.apply(DeltaLog.scala:275)
        at org.apache.spark.sql.delta.DeltaLog$$anonfun$org$apache$spark$sql$delta$DeltaLog$$updateInternal$1.apply(DeltaLog.scala:275)
        at com.databricks.spark.util.DatabricksLogging$class.recordOperation(DatabricksLogging.scala:77)
        at org.apache.spark.sql.delta.DeltaLog.recordOperation(DeltaLog.scala:58)
        at org.apache.spark.sql.delta.metering.DeltaLogging$class.recordDeltaOperation(DeltaLogging.scala:103)
        at org.apache.spark.sql.delta.DeltaLog.recordDeltaOperation(DeltaLog.scala:58)
        at org.apache.spark.sql.delta.DeltaLog.org$apache$spark$sql$delta$DeltaLog$$updateInternal(DeltaLog.scala:274)
        at org.apache.spark.sql.delta.DeltaLog$$anonfun$update$2.apply(DeltaLog.scala:235)
        at org.apache.spark.sql.delta.DeltaLog$$anonfun$update$2.apply(DeltaLog.scala:235)
        at org.apache.spark.sql.delta.DeltaLog.lockInterruptibly(DeltaLog.scala:207)
        at org.apache.spark.sql.delta.DeltaLog.update(DeltaLog.scala:234)
        at org.apache.spark.sql.delta.DeltaHelper$.loadDeltaLatestSnapshot(DeltaHelper.scala:178)
        at io.delta.hive.DeltaStorageHandler.preCreateTable(DeltaStorageHandler.scala:189)
        at org.apache.hadoop.hive.metastore.HiveMetaStoreClient.createTable(HiveMetaStoreClient.java:747)
        at org.apache.hadoop.hive.metastore.HiveMetaStoreClient.createTable(HiveMetaStoreClient.java:740)
        at sun.reflect.GeneratedMethodAccessor290.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.hadoop.hive.metastore.RetryingMetaStoreClient.invoke(RetryingMetaStoreClient.java:173)
        ... 29 more
```

**Comment by chongchuanbing** [comment]
solution:
change the spark config in hive. add config "fs.hdfs.impl.disable.cache=true"

suggest open the config of spark in hive.

```
def spark: SparkSession = {
    // TODO Configure `spark` to pick up the right Hadoop configuration.
    if (System.getProperty("delta.log.cacheSize") == null) {
      System.setProperty("delta.log.cacheSize", "1")
    }
    val sparkSession = SparkSession.builder()
      .master("local[*]")
      .appName("Delta Connector")
      .config("spark.ui.enabled", "false")
      .getOrCreate()
    // Trigger codes that add `SparkListener`s before stopping the listener bus. Otherwise, they
    // would fail to add `SparkListener`s.
    sparkSession.sharedState
    sparkSession.sessionState
    sparkSession.sparkContext.listenerBus.stop()
    sparkSession
  }
```

**Comment by zsxwing** [comment]
This has been resolved by #51 which removes Spark from the Hive connector.
